components/server/src/user/user-service.ts

```diff
@@ -355,17 +355,6 @@ export class UserService {
     }
 
     async onAfterUserLoad(user: User): Promise<User> {
-        try {
```

Comment:
Nice, I like this approach much better as it makes it explicitly user trigerred and removes "special-cases" like this